styleguide/components/31-molecules/contact-block/contact-block.config.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+  title: 'Contact block',
+  default: 'contact-block-with-phone',
+  status: 'alpha',
+  handle: 'contact-block',
+  collated: true,
+  collator: function (markup, item) {
+    return `<!-- Start: @${item.handle} -->\n<dt>${item.name}</dt><dd>${markup}</dd>\n<!-- End: @${item.handle} -->\n`;
+  },
```

Comment:
Missing preview property

Reply:
It has a default right? No need to specify this if it's the default?

Reply:
Yes, but we don't want the default in a collation. We want the description list.

Reply:
Fixed.